client/react/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Blog</title>
+    <title>BeOk</title>
```

Comment:
I wonder if the "k" should be capitalized... what do you think? 😛

Reply:
hm, like it!